dist/utils/PropExample.js

```diff
@@ -52,7 +52,7 @@ var Definition = function (_PureComponent) {
           { className: 'row' },
```

Comment:
We should consider splitting out utils into two types...ones that we export and ones that we don't.